tests/wpunit/Tribe/Tickets/__snapshots__/RSVPTest__it_should_render_rsvp_step with data set ari__1.php

```diff
@@ -108,7 +108,7 @@ class="tribe-tickets__rsvp-ar-guest-list-item-button tribe-tickets__rsvp-ar-gues
 		<div class="tribe-tickets__rsvp-ar-form">
 
 	<input type="hidden" name="tribe_tickets[TICKET_ID][ticket_id]" value="TICKET_ID">
-	<input type="hidden" name="tribe_tickets[TICKET_ID][attendees][0][order_status]" value="not-going">
+	<input type="hidden" name="tribe_tickets[TICKET_ID][attendees][0][order_status]" value="">
```

Comment:
Since the original snapshot had the value as "not-going" should the default for going be "not-going" instead of empty ""?

Reply:
Yeah. That's the one I updated with Slic. I think its not supposed to have "not-going" @juanfra